index.html

```diff
@@ -76,6 +76,202 @@ <h4 class="title">I am a Software Developer</h4>
+                <div class="bg-img">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button>See Projects</button>
+                </div>
+            </div>
+            <div class="grid-items pj">
+                <div class="bg-img">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button>See Projects</button>
+                </div>
+            </div>
+            <div class="grid-items pj">
+                <div class="bg-img">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button>See Projects</button>
+                </div>
+            </div>
+            <div class="grid-items pj">
+                <div class="bg-img">
+                    <h3>Profesional Art Printing Data</h3>
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    <button>See Projects</button>
+                </div>
+            </div>
+        </div>
+        <!-- end of my my recent works -->
+
+        <!-- about me -->
+        <section class="about-me">
+            <div class="about-content">
+                <h1 class="about-heading">About me</h1>
+                <p class="about-text">
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <button class="get-my-resume" type="button">Get My Resume</button>
```

Comment:
- [ ] Kindly implement the active, hover, and pressed styling for this button as specified in the template design 👇 
![image](https://user-images.githubusercontent.com/46839548/153714518-60bcc92d-9418-497a-a2a5-bbd8d78970cd.png)